hardhat.config.ts

```diff
@@ -81,9 +83,7 @@ const config: HardhatUserConfig = {
     mainnet: {
       chainId: 1,
       url: MAINNET_RPC_URL,
-      accounts: {
-        mnemonic: MNEMONIC,
-      },
+      accounts: [DEPLOYER_PK],
```

Comment:
why add this DEPLOYER_PK approach? trying to separate out keys used for different purposes?

Reply:
maybe you didn't mean to check this in?

Reply:
yeah i generally prefer to use a pk, as it is less surface area for moving sensitive info around. if i accidentally check in the seed then that entire wallet is compromised (not that i'm putting anything significant on these addresses anyways.
i'll roll it back